org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -156,6 +157,12 @@ public class TargetConfig {
      */
     public boolean fastMode = false;
```

Comment:
Check where `fastMode` is set to true, and if only at places where `fastModeASTNode` can be specified, maybe remove this.

Reply:
Maybe make a separate issue/PR, this property has at least 8 usages.